src/has.ts

```diff
@@ -20,6 +29,12 @@ export const testCache: { [feature: string]: FeatureTestResult } = {};
  */
 export const testFunctions: { [feature: string]: FeatureTest } = {};
 
+/**
+ * A cache of unresolved thenables (probably promises)
+ * @type {{}}
+ */
+export const testThenables: { [feature: string]: FeatureTestThenable} = {};
```

Comment:
I guess we don't. The other caches are exported so I guess I assumed that this one needs to be too. Upon looking at it though it looks like the other ones are exported for the unit tests, but we're not using `testThenables` in the unit tests directly.